models/DataObject/ClassDefinition/Data/ImageGallery.php

```diff
@@ -345,15 +311,15 @@ public function getDataForEditmode($data, $object = null, $params = [])
     }
 
     /**
-     * @see Data::getDataFromEditmode
-     *
-     * @param array|null $data
+     * @param mixed $data
      * @param null|DataObject\Concrete $object
-     * @param mixed $params
+     * @param array $params
```

Comment:
since that change (from mixed to array) is quite often in the class definition data objects ... should we apply that already to 10.5?

Reply:
Will be done with https://github.com/pimcore/pimcore/issues/13577